pkgs/gamescope-session/environment.patch

```diff
@@ -1,16 +1,29 @@
 diff --git a/gamescope-session b/gamescope-session
-index 7101e70..2ff1bd5 100755
+index 1fea3be..b41e819 100755
 --- a/gamescope-session
 +++ b/gamescope-session
-@@ -243,6 +243,11 @@ else
-        echo >&2 "!! Failed to claim global gamescope stats session"
+@@ -245,6 +245,11 @@ else
+ 	echo >&2 "!! Failed to claim global gamescope stats session"
  fi
-
+ 
 +# Jovian NixOS environment overrides hook
 +if test -e /etc/xdg/gamescope-session/environment; then
 +       . /etc/xdg/gamescope-session/environment;
 +fi
 +
  read_gamescope_env() {
-        if read -r -t 3 response_x_display response_wl_display <> "$socket"; then
-                export DISPLAY="$response_x_display"
+ 	if read -r -t 3 response_x_display response_wl_display <> "$socket"; then
+ 		export DISPLAY="$response_x_display"
+diff --git a/steam-launcher b/steam-launcher
+index d574495..d205026 100644
+--- a/steam-launcher
++++ b/steam-launcher
+@@ -1,6 +1,7 @@
+ #!/bin/bash
+ 
+-steamargs=("-steamos3" "-steampal" "-steamdeck" "-gamepadui")
++# Jovian: remove -steamdeck here so we can set it in the wrapper instead
++steamargs=("-steamos3" "-gamepadui")
```

Comment:
Should we yeet them all out and add them all in the package instead? I don't know if there's any reason to forcibly keep these around.

Reply:
Right, I forgot this is ran within the session script...